http-gateway/web/src/containers/configuration/configuration.js

```diff
@@ -0,0 +1,25 @@
+import { useIntl } from 'react-intl'
+
+import { Layout } from '@/components/layout'
+import { messages as menuT } from '@/components/menu/menu-i18n'
```

Comment:
could just be shortened to t.

Reply:
For convention I only use `t` for the messages which belongs to the component with the same name. Since these are the messages of the Menu, I use the name `menuT` instead.